interface/modules/custom_modules/oe-module-faxsms/templates/phone_widget.html.twig

```diff
@@ -0,0 +1,120 @@
+{# phone_widget.html.twig #}
+
+<div id="rc-container" class="d-flex flex-row align-items-center justify-content-between">
+    <div id="rc-header">
+        <script>
+            const RC_CLIENT_ID = {{ clientId|js_url }};
+            const RC_CLIENT_SECRET = {{ clientSecret|js_url }};
+            const RC_JWT = {{ jwt|js_url }};
```

Comment:
would remove the url component of escaping since you are escaping url correctly below (otherwise you will double url escape these values)